docker/README.md

```diff
@@ -36,9 +32,9 @@ Usage: bash run.sh --traindata=<path> --testdata=<path> [--gpus=GPU_IDs] [--shar
 
 > `2222` port is exposed by docker which is mapped to `8888` for jupyter notebook server within docker.
 
-> optionally one can pass GPU IDs by using the tag `--gpus`. By default, two GPUS `0,1` will be passed to the docker container.
+> optionally one can pass GPU IDs by using the tag `--gpus`. By default, four GPUs `0,1,2,3` will be passed to the docker container. In order to simulate our training environment, we recommend you use 4 GPUs.
 
-> `--weights` option is only necessary to skip the traning steps and generate submissions directly from our models. This option should point to the path `<path>/weights` where the weights (`.pth` files) downloaded from `our_weights` are located.
+> `--weights` specifies where you want to save/load the model weights. For loading our trained weights, the option should point to the path `<path>/weights` where the weights (`.pth` files) located in `our_weights` are located. By default, the model saves/loads weights in a subdir `weights` in the current working directory.
```

Comment:
remove `are located.`

Reply:
it clarifies the full functionality though

Reply:
dude. `the option should point to the path `<path>/weights` where the weights (`.pth` files) located in `our_weights` are located. ` does this sound right to you? I just requested you to rephrase. You don't have to defend every single sentence that you speak to the end of the world.

Reply:
i missed this. I can change `located in our_weights are located` to `downloaded from our_weights are located` if that's what you are asking.